#!/usr/bin/env python
PACKAGE = "dataAssoc_experimental"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("laser_height", double_t, 0, "The height of the laser from ground", 0.8, .1, 5)
gen.add("pixel_padding", int_t, 0, "Number of pixel to pad around the ROI", 18, 1, 100)
gen.add("color_cost", double_t, 0, "The parameter of cost function for color", 20, 0, 100)
gen.add("dist_cost", double_t, 0, "The parameter of cost function for distance", 1.0, 0, 20)
gen.add("cost_threhold", double_t, 0, "The threshold value for the cost function used when performing data association", 1.5, 0, 20)
gen.add("merge_dist", double_t, 0, "The distance value for any pedestrian in view to be considered as a merged cluster", 0.5, 0, 2) 
exit(gen.generate(PACKAGE, "dataAssoc_experimental", "CameraParam"))